File: flut/.flutter/lib/flutter/rendering/wrap.dart
import 'package:flutter/material.dart';
import 'package:flut/flut/object.dart';

class FlutWrapAlignment extends FlutEnumObject<WrapAlignment> {
  const FlutWrapAlignment()
    : super('WrapAlignment', const {
        'start': WrapAlignment.start,
        'end': WrapAlignment.end,
        'center': WrapAlignment.center,
        'spaceBetween': WrapAlignment.spaceBetween,
        'spaceAround': WrapAlignment.spaceAround,
        'spaceEvenly': WrapAlignment.spaceEvenly,
      });
}

class FlutWrapCrossAlignment extends FlutEnumObject<WrapCrossAlignment> {
  const FlutWrapCrossAlignment()
    : super('WrapCrossAlignment', const {
        'start': WrapCrossAlignment.start,
        'end': WrapCrossAlignment.end,
        'center': WrapCrossAlignment.center,
      });
}

File: flut/flutter/rendering/wrap.py
from flut._flut.engine import FlutEnumObject


class WrapAlignment(FlutEnumObject):
    start: "WrapAlignment"
    end: "WrapAlignment"
    center: "WrapAlignment"
    spaceBetween: "WrapAlignment"
    spaceAround: "WrapAlignment"
    spaceEvenly: "WrapAlignment"


class WrapCrossAlignment(FlutEnumObject):
    start: "WrapCrossAlignment"
    end: "WrapCrossAlignment"
    center: "WrapCrossAlignment"
